include/picongpu/traits/attribute/GetMass.hpp

```diff
@@ -34,7 +34,8 @@ namespace picongpu
              *
              * @param weighting weighting of the particle
              * @param particle a reference to a particle
-             * @return mass of the makro particle
+             *
+             * @return mass of the macro particle, UNIT_MASS
```

Comment:
QUESTION: Why that addition? Is this pertinent?

Reply:
I changed the GetMass documentation to fit to the documentation style used for atomicPhyisc returns as it is used repeatedly in the atomcPhysics,  and while I was at it I fixed the typo.

Could be moved to separate PR, or I removed I am open to both.

Reply:
In general the problem I ran into is that atomicPhysics needs a different normalization than the PIC-loop, since
atomicPhysics deals with fundamentally much smaller quantities and therefore I had to sometime use different units than the main code to avoid numerical errors.

Consequently I had to document what the unit the return values of some of my functions were and to make less errors in the internal unit conversions I created documentation to keep track of units.

Reply:
@chillenzer not strongly necessary but okisch.